sdcm/utils/monitorstack.py

```diff
@@ -211,7 +211,7 @@ def get_monitoring_stack_scylla_version(monitoring_stack_dir):
         with open(os.path.join(monitoring_stack_dir, 'monitor_version'), 'r') as f:  # pylint: disable=invalid-name
             versions = f.read().strip()
         monitoring_version, scylla_version = versions.split(':')
-        return monitoring_version, scylla_version
+        return monitoring_version, "all"
```

Comment:
how is this related to this PR? What is the purpose/reason for this change?

Reply:
It's not, I've noticed that when we use `show-monitor` we don't have the other dashboards, since scylla version comes out as 4.4 on master, and we don't have a versions.

I think we could just put all of the versions available in scylla-monitoring, in case we need them.

Reply:
I'll take it to a different PR

Reply:
after a closer look, 
the problem was in the way we handle the version, took it to this PR:
https://github.com/scylladb/scylla-cluster-tests/pull/2847